docs/specifications/tests/Delegation-TP/delegation01.md

```diff
@@ -1,49 +1,130 @@
-## DELEGATION01: Minimum number of name servers   
+# DELEGATION01: Minimum number of name servers   
 
-### Test case identifier
+## Test case identifier
 
-**DELEGATION01:** Minimum number of name servers
+**DELEGATION01**
 
-### Objective
+## Objective
 
-Section 4.1 of [RFC 1034](https://tools.ietf.org/html/rfc1034) specifies that
-there must be at least minimum two name servers for a domain. This test is
-done to verify this condition
+Section 4.1 of [RFC 1034] specifies that there must be a minimum of two name servers 
+for a domain. This test is done to verify this condition.
 
-### Inputs
+The RFC ([RFC 1034]) predates IPv6. Since IPv4 and IPv6 work as separate networks, this
+test case has been extended to test for two name servers that resolve into IPv4 addresses
+and IPv6 addresses respectively.
 
-The domain name to be tested.
 
-### Ordered description of steps to be taken to execute the test case
+## Inputs
 
-1. Obtain the complete set of name servers from parent using
-   [Method2](../Methods.md).
-2. If the number of name servers (distinct NS records) returned in previous step is less than two, the
-   record as an error.
-3. Obtain the complete set of name servers from the child 
-   using [Method3](../Methods.md).
-4. If the number of name servers (distinct NS records) returned in the previous step is less than two, the
-   record as an error.
-5. If an error was recorded in step 2 or step 4, then the test case fails.
- 
-### Outcome(s)
+The domain name to be tested ("Child Zone").
 
-If the total amount of name servers used for the domain are two or more,
-then the test case succeeds.
+## Ordered description of steps to be taken to execute the test case
 
-### Special procedural requirements
+ 1. Using [Method2], obtain the complete set of names of the name servers 
+    from parent for the *Child Zone* (distinct NS records).
+
+ 2. Count the name server names:
+    1. If zero or one, emit *NOT_ENOUGH_NS_PARENT*.
+    2. If two or more, emit *ENOUGH_NS_PARENT*.
+
+ 3. Obtain the IP addresses for the set of name servers from the parent for the 
+    *Child Zone*.
```

Comment:
How's this to be done? Is Method3 the best way?

Reply:
Yes, will update.